### ISSUE TITLE
import_annotations: shift onsets/offsets so that time_seek = 0 ?

### ISSUE THREAD

**Issue by lucasgautheron** [LAAC-LSCP/ChildProject#138]
Eeach annotation file has a `time_seek` column, which is the shift between the times as expressed in the annotation and the actual corresponding times of the recording.

Currently we can have negative time_seek e.g.: https://github.com/LAAC-LSCP/alphen-data/blob/master/metadata/annotations.csv. Not really a problem per se, unless maybe aesthetically. 

I was thinking, one might want to shift all the segment_onset/segment_offset and range_onset/range_offset in such a way that time_seek could be set to 0, which is very simple to do

In:
```python
    def import_annotations(self, input: pd.DataFrame, threads: int = -1, import_function: Callable[[str], pd.DataFrame] = None):
```

The `input` dataframe could have a 'cancel_time_seek' column, which if set to True, would shift all times so that time_seek can be set to zero.

However, there might be a good reason not to do that : having the time_seek stored somewhere might be helpful in case someone wants to work with the raw annotations directly, then they know how to relate timestamps from the raw file to the audio. 

In other words: we should make it clear that time_seek should always be taken into account, and that it can be negative. I don't think shifting is a good idea, though it may seem attractive.

**Comment by sarpu** [comment]
I second this, and I think it would be good to quote your definition of `time_seek` here verbatim in the docs/package. I couldn't quite gather what it was from the available docs but this provides great clarification.
